ocaml/xapi/repository.ml

```diff
@@ -128,9 +128,11 @@ let sync ~__context ~self ~token ~token_id =
     write_initial_yum_config () ;
     clean_yum_cache repo_name ;
     (* Remove imported YUM repository GPG key *)
-    Xapi_stdext_unix.Unixext.rm_rec (get_repo_config repo_name "gpgdir") ;
+    if Pkg_mgr.(active () = Yum) then
```

Comment:
Generally, I think this `if` could be wrapped into `Pkg_mgr.<function>`.

Reply:
Ideally it can, but our `Pkg_mgr` module just dispatch command line and arguments as we talked.
Otherwise, providing a interface, like `clean_gpg_dir` while other interface just parse arguments sounds strange.